django-verdant/rca/templates/rca/home_page_2018.html

```diff
@@ -5,31 +5,54 @@
     <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css"/>
 {% endblock %}
 
+{% block body_class %}{{ block.super }} template-home-page{% endblock %}
+
+{% block hero %}
+    {% image self.hero_image original as hero_image %}
+    <section class="hero" style="background-image: linear-gradient(to right, rgba(0, 0, 0, 0.6) 0%, rgba(0, 0, 0, 0) 100%), url('{{ hero_image.url }}');">
+        <div class="hero__inner page-wrapper">
+            <a href="{{ header_navigation_links.navigation_link_1_url }}">{{ header_navigation_links.navigation_link_1_text }}</a>
+            <a href="{{ header_navigation_links.navigation_link_2_url }}">{{ header_navigation_links.navigation_link_2_text }}</a>
+
+            <p class="hero__text">{{ self.hero_text }}</p>
+
+            <div class="hero__title-wrapper">
+                <h2 class="hero__title">Study with us</h2>
+
+                {% if last_viewed_programme %}
+                    <h3>Recently viewed: <a href="{% pageurl last_viewed_programme %}">{{ last_viewed_programme }}</a></h3>
+                {% endif %}
+
+                {# Browse all programmes link needs a ProgrammeFinderPage to exist #}
+                {% if programme_finder_page_url %}
+                    <a class="hero__browse-link" href="{{ programme_finder_page_url }}">
+                        <span class="hero__browse-text">
+                            <span class="hero__browse-text--text-alt">Browse</span> our programmes
+                        </span>
+
+                        <svg class="hero__browse-icon">
+                            <use xlink:href="#arrow-thin"></use>
+                        </svg>
+                    </a>
+                {% endif %}
+            </div>
+
+            <div class="hero__search-wrapper">
+                <label class="hidden" for="course_search">Search our programmes: enter a title or keyword</label>
+
+                <input class="hero__search-input" type="text" name="course_search" id="course_search" placeholder="SEARCH OUR PROGRAMMES: ENTER A TITLE OR KEYWORD">
+
+                <svg class="hero__search-icon">
+                    <use xlink:href="#search"></use>
```

Comment:
add title

Reply:
aslo ought to be a button element - won't be clickable in a screen reader.

Reply:
there's not actually anything that needs to be clicked, results show up as you type, I'm not sure how that impacts accessibility

Reply:
Might be worth a quick investigation - check if the ui-autocomplete library mentions this.